vmdb/spec/models/ems_refresh/refreshers/foreman_refresher_spec.rb

```diff
@@ -134,6 +131,29 @@ def assert_configured_system
     expect(system.configuration_profile).to   eq(child)
   end
 
+  it "loads data with locations" do
```

Comment:
but we need to serialize it to the database
and we need to test that it is properly saving to the database

so in this case, not sure how to do this